### PR TITLE
Regional analysis percentiles

### PR DESCRIPTION
**Pull request by mattwigway**
Allow setting the percentile of a regional analysis. Depends on conveyal/analysis-backend#27.

### CODE REVIEW

File: configurations/default/messages.yml [hunk 1]
```diff
@@ -104,6 +105,7 @@ analysis:
   errorsInScenario: Errors were found in the scenario:
   warningsInScenario: Warnings were found applying the scenario
   errorsInModification: in modification %s
+  travelTimePercentile: Travel time percentile (reliab)
```
Comment (mattwigway) [inline configurations/default/messages.yml]: Nope, not abbreviated on purpose, oops.

Comment (mattwigway) [inline configurations/default/messages.yml]: Fixed, I'm not sure why we'd want to move this to the top level though.

Comment (trevorgerhardt) [inline configurations/default/messages.yml]: No reason now! Just need to update the tests then.

**Comment by mattwigway** [comment]
@trevorgerhardt I think this is done, turns out the failing tests were showing an issue in the code.

**Pull request merged**